client/src/common/components/Members/components/MemberDetails.jsx

```diff
@@ -369,9 +367,9 @@ MemberDetails.propTypes = {
   isGuest: PropTypes.bool,
   isMember: PropTypes.bool,
   isOwner: PropTypes.bool,
-  isPrivate: PropTypes.bool,
   match: RouterMatchPropType.isRequired,
   route: PropTypes.string,
+  type: PropTypes.string,
```

Comment:
shouldn't this be required?

Reply:
type shouldn't be required because MemberDetails component is used in cohort also, and cohorts don't have type property